source/braille.py

```diff
@@ -3132,18 +3132,17 @@ def handlePostConfigProfileSwitch(self):
 
 		if (configuredTether := config.conf["braille"]["tetherTo"]) != TetherTo.AUTO.value:
 			self._tether = configuredTether
-		if config.conf["braille"]["translationTable"] == "auto":
-			config.conf["braille"]["translationTable"] = brailleTables.getDefaultTableForCurLang(
-				brailleTables.TableType.OUTPUT,
-			)
 		tableName = config.conf["braille"]["translationTable"]
 		# #6140: Migrate to new table names as smoothly as possible.
 		newTableName = brailleTables.RENAMED_TABLES.get(tableName)
 		if newTableName:
 			tableName = config.conf["braille"]["translationTable"] = newTableName
 		if tableName != self._table.fileName:
 			try:
-				self._table = brailleTables.getTable(tableName)
+				if config.conf["braille"]["translationTable"] == "auto":
+					self._table = brailleTables.getDefaultTableForCurLang(brailleTables.TableType.OUTPUT)
```

Comment:
Is this correct? Shouldn't this be?
```suggestion
					tableName = brailleTables.getDefaultTableForCurLang(brailleTables.TableType.OUTPUT)
```
That means instead of the else block, you can just load tableName.